build.gradle

```diff
@@ -267,6 +267,31 @@ task clean {
     dependsOn cleanLocalMavenRepos
 }
 
+task manualClean {
+    description = 'Clean build files manually, which means that this task works without clean tasks'
+    group = 'Clean'
+
+    doLast {
+        // clean 'build' directories
+        exec {
+            workingDir "${rootDir}"
+            commandLine 'find', '.', '-type', 'd', '-name', 'build', '-print', '-exec', 'rm', '-rf', '{}', ';', '-prune'
```

Comment:
I didn't know `-delete` option.

Then I tried `find . -type d -name build -delete` , but `build` directories are not deleted.
What's wrong?